### PR TITLE
breadcrumbs: uninstall_wrongly_reports_cask_as_not_installed

### PR DESCRIPTION
**Pull request by vitorgalvao**

### CODE REVIEW

File: doc/reporting_bugs/uninstall_wrongly_reports_cask_as_not_installed.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<sup><sub>[Go back](../../README.md#reporting-bugs)</sup></sub>
+
+# `uninstall` wrongly reports cask as not installed
+
+Reports of this bug are usually accompanied with the output of `brew cask list` showing the cask is indeed installed. [You should not relly on `brew cask list`](https://github.com/caskroom/homebrew-cask/issues/14058) anyway, and this error is somewhat related to it.
```
Comment (jawshooah) [inline doc/reporting_bugs/uninstall_wrongly_reports_cask_as_not_installed.md]: s/relly/rely/

Comment (vitorgalvao) [inline doc/reporting_bugs/uninstall_wrongly_reports_cask_as_not_installed.md]: Thank you. Fixed.

File: doc/reporting_bugs/uninstall_wrongly_reports_cask_as_not_installed.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<sup><sub>[Go back](../../README.md#reporting-bugs)</sup></sub>
+
+# `uninstall` wrongly reports cask as not installed
+
+Reports of this bug are usually accompanied with the output of `brew cask list` showing the cask is indeed installed. [You should not rely on `brew cask list`](https://github.com/caskroom/homebrew-cask/issues/14058) anyway, and this error is somewhat related to it.
+
+This is prone to happen when you try to uninstall a cask that has been updated since you last installed, or when a cask no longer exists in the repo. We’re working to handle those situations better.
```
Comment (adidalal) [inline doc/reporting_bugs/uninstall_wrongly_reports_cask_as_not_installed.md]: Possibly add a note about how such Casks show up with `(!)` next to them in `brew cask list`

Comment (vitorgalvao) [inline doc/reporting_bugs/uninstall_wrongly_reports_cask_as_not_installed.md]: Done.

**Comment by adidalal** [comment]
LGTM

**Pull request merged**